src/index.html

```diff
@@ -1,17 +1,113 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:3000/api" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form_field field">
+          Surname:
+          <input type="text" name="usersurname"
+          autocomplete="off" required>
```

Comment:
```suggestion
          <input 
            type="text" 
            name="usersurname"
            autocomplete="off" 
            required
          >
```

проверь стиль кода везде

Reply:
Исправил